php.ini-production

```diff
@@ -1764,20 +1764,20 @@ ldap.max_links = -1
 
 [opcache]
 ; Determines if Zend OPCache is enabled
-;opcache.enable=0
+;opcache.enable=1
 
 ; Determines if Zend OPCache is enabled for the CLI version of PHP
-;opcache.enable_cli=0
+;opcache.enable_cli=1
```

Comment:
An aspect of this merge caused us a bit of a headache while upgrading to PHP 7.1. It might be worth documenting this change here:

http://php.net/manual/en/migration71.other-changes.php

In our case we have a very large codebase and a great deal of very short lived PHP processes running on the CLI. The extra overhead of generating, and then discarding (as we are not saving it into a file cache as per these new default arguments) the opcache was bringing our server down (becoming CPU bound very quickly). I don't see the point in enabling 'opcache.enable_cli' by default without the file cache as the opcache will need to be regenerated on the next execution? This might be OK for certain types of CLI application, but for us it was an issue.

A mention in the 'upgrade notes' would just give people a pointer that this change might affect them.